ndohyep/urls.py

```diff
@@ -23,6 +23,7 @@
     url(r'commenting/reply/(?P<parent_id>\d+)/$', CommentReplyForm.as_view(),
         name='comments-reply'),
     url(r'', include(wagtail_urls)),
+    url(r'^djga/', include('google_analytics.urls')),
```

Comment:
I think we still need to reference the image that does the analytics tracking from the templates somewhere.